src/components/Search/SearchContact.js

```diff
@@ -27,7 +27,7 @@ const Search = ({animatedValue, text, setText}) => {
 const styles = StyleSheet.create({
   wrapperSearch: {
     flex: 1,
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.lightgrey` to `COLORS.gray100` for the `backgroundColor` property in the `wrapperSearch` style seems fine, as long as it aligns with the new color scheme you're implementing. Ensure that this color change doesn't affect readability or accessibility (contrast) of any overlaid text or elements.
```diff
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->